docs/plugins/transform-regexp-constructors.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: docs
+title: Regexp constructors transform
+description: This plugin allows Babel to transform boolean literals into !0 for true and !1 for false
```

Comment:
The description should be `This changes RegExp constructors into literals if the RegExp arguments are strings.`